openlibrary/plugins/openlibrary/home.py

```diff
@@ -106,18 +106,28 @@ def GET(self):
         return web.template.TemplateResult(cached_homepage)
 
 
+def get_random_borrowable_ebook_keys(count: int) -> list[str]:
```

Comment:
Can we switch this to use the decorator? I find that makes the code easier to digest.